vspace/src/main/webapp/WEB-INF/views/exhibition/space.html

```diff
@@ -66,26 +81,66 @@
         font-size: 20px;
     }
 }
+
+.wrapper {
+    float: left;
+}
+
+.spaceClass {
+    position: relative;
+    right: 5%;
+    left: 5%;
+}
+
+#sidebar {
+    min-width: 250px;
+    max-width: 250px;
+    top: -40px;
+    left: 0;
+    /* top layer */
+    z-index: 9999;
+    transition: all 0.5s;
+}
+
+#sidebar.active {
+    margin-left: -250px;
+}
+
+.btn {
+    padding: 10px;
+}
+
+#dismiss {
+    width: 35px;
+    height: 35px;
+    position: absolute;
+    /* top right corner of the sidebar */
+    top: 110px;
+    right: 1000px;
+}
+
+@media ( max-width : 768px) {
+    #sidebar {
+        margin-left: -250px;
+    }
+    #sidebar.active {
+        margin-left: 0;
+    }
+}
 </style>
 </head>
 
 <body>
     <div layout:fragment="content">
         <script th:inline="javascript">
 		//# sourceURL=click.js
+
 		let maxWidth='(max-width: 1200px)';
-		function openNav(){
-            if (window.matchMedia(maxWidth).matches){
-                $("#mySidenav").css("width", "170px");
-                $("#mySidenav").css("height", "260px");
-            }
-		    else{
-		        $("#mySidenav").css("width", "240px");
-		        $("#mySidenav").css("height", "[(${display.height})]px");
-            }
+		function openNav(){	
+			document.getElementById("mySidenav").style.width = "250px";
         }
-		function closeNav(){
-		    $("#mySidenav").css("width", "0px");
+        function closeNav(){
+        	document.getElementById("mySidenav").style.width = "0";
```

Comment:
Is there a reason you replaced the jquery with regular javascript?

Reply:
This code is not added by me. Maybe while indentation and code formatting changes, it appears to have been added by me.

Reply:
a) it shouldn't show up as changed and b) clean code rule #X, leave the code better than you find it. ;)